.config/jest/mocks/grafana-llm.ts

```diff
@@ -0,0 +1,12 @@
+// Mock implementation of @grafana/llm for testing
+// This avoids ES module import issues in Jest
+
+export const llm = {
+  Model: {
+    BASE: 'base',
+  },
+  enabled: jest.fn().mockResolvedValue(true),
+  chatCompletions: jest.fn().mockResolvedValue({
+    choices: [{ message: { content: 'test' } }],
+  }),
+};
```

Comment:
Just a curious question, do we need to add these overrides in the `.config` folder? Seems like they could go away with future updates?